pos_picking_load/tests/test_pos_picking_load.py

```diff
@@ -75,7 +79,9 @@ def test_pos_picking_load(self):
             [('origin_picking_id', 'in', sale_order.picking_ids.ids)])
         self.assertTrue(pos_order)
         self.assertEqual(pos_order.origin_picking_id.state, 'cancel')
-        self.assertIn(pos_order.picking_id, sale_order.picking_ids)
+        # self.assertIn(pos_order.picking_id, sale_order.picking_ids)
```

Comment:
Here too. I think if it is not valid anymore it is better to just remove it then commenting it out.